src/Infrastructure/LeanCode.Mixpanel/MixpanelAnalytics.cs

```diff
@@ -15,81 +17,103 @@ public class MixpanelAnalytics
     {
         private readonly Serilog.ILogger logger = Serilog.Log.ForContext<MixpanelAnalytics>();
 
-        private readonly MixpanelHttpClient client;
+        private readonly HttpClient client;
         private readonly MixpanelConfiguration configuration;
 
-        public MixpanelAnalytics(MixpanelHttpClient client, MixpanelConfiguration configuration)
+        public MixpanelAnalytics(HttpClient client, MixpanelConfiguration configuration)
         {
             this.configuration = configuration;
             this.client = client;
         }
 
-        public Task AliasAsync(string newId, string oldId)
+        public Task AliasAsync(string newId, string oldId, CancellationToken cancellationToken = default)
         {
             return TrackAsync(newId, "$create_alias", new Dictionary<string, object>()
             {
                 ["distinct_id"] = oldId,
                 ["alias"] = newId,
-            });
+            }, cancellationToken: cancellationToken);
         }
 
-        public Task TrackAsync(string userId, IMixpanelEvent mixpanelEvent, bool isImport = false)
-            => TrackAsync(userId, mixpanelEvent.EventName, mixpanelEvent.Properties, isImport);
+        public Task TrackAsync(
+            string userId,
+            IMixpanelEvent mixpanelEvent,
+            bool isImport = false,
+            CancellationToken cancellationToken = default)
+            => TrackAsync(userId, mixpanelEvent.EventName, mixpanelEvent.Properties, isImport, cancellationToken);
```

Comment:
`{ return …; }` here and in a few other places in this file.

Reply:
In this file I'd keep it as is.